pkg/build/api/types.go

```diff
@@ -475,29 +475,34 @@ const (
 	// build pod but one was already present.
 	StatusReasonBuildPodExists StatusReason = "BuildPodExists"
 
+	// StatusReasonUnresolveableEnvironmentVariable indicates that an error occurred processing
+	// the supplied options for environment variables in the build strategy environment
+	StatusReasonUnresolveableEnvironmentVariable StatusReason = "UnresolvableEnvironmentVariable"
```

Comment:
https://www.merriam-webster.com/dictionary/unresolvable  (also you spelled it inconsistently here... :) )

(and yes i'm pretty sure my suggestion spelled it wrong too)